keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -104,10 +178,40 @@ Currently, workloads within a ClusterQueue are sorted based on 1. Priority and 2
 
 #### Proposed Sorting
 
-The `pkg/workload` package could be modified to include a conditional (`if evictionReason == kueue.WorkloadEvictedByPodsReadyTimeout`) that controls which timestamp to return based on the configured ordering strategy. The same sorting logic would also be used when sorting the heads of queues.
+The `pkg/workload` package could be modified to include a conditional (`if evictionReason == kueue.WorkloadEvictedByPodsReadyTimeout`) 
+that controls which timestamp to return based on the configured ordering strategy.
+The same sorting logic would also be used when sorting the heads of queues.
 
 Update the `apis/config/<version>` package to include `Creation` and `Eviction` constants.
 
+### Exponential Backoff Mechanism
+
+When the kueueConfig `backOffLimitCount` or `backOffLimitTimeout` is set and there are evicted workloads by waitForPodsReady,
+the queueManager returns evicted workloads that an exponential backoff duration finished and other workloads as a headWorkloads.
+
+The queueManager calculates an exponential backoff duration by [the Step function](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait@v0.29.1#Backoff.Step).
+
+### Evaluation of BackOffLimits
+
+#### backOffLimitCount
+
+When a workload eviction is issued with `PodsReadyTimeout` condition, 
+a workload `.status.requeuedCount` is incremented by 1 each time　in the workload controller.
+
+After that, when a workload `.status.requeudCount` reaches the kueueConfig `.waitForPodsReady.requeueingStrategy.backOffLimitCount`,
+a workload is deactivated by setting false to `.spec.active` instead of be suspended in the jobframework reconciler.
+
+#### backOffLimitTimeout
```

Comment:
What is the use case here?

Why isn't the backoffLimitCount enough?

Reply:
> Why isn't the backoffLimitCount enough?

Because `backoffCount` is difficult to estimate how many hours jobs will actually be retrying. So, to improve the possibility for estimation, I added timeout separate from count.

If you think that we shouldn't have `backoffLimitTimeout` separate from `backoffLimitCount`, I'm ok with removing `backoffLimitTimeout`.

@alculquicondor WDYT?

Reply:
SGTM
We can start only support `backoffCount` for now.

Reply:
Done.